packages/mold.rb

```diff
@@ -6,18 +6,22 @@
 class Mold < Package
   description 'A Modern Linker'
   homepage 'https://github.com/rui314/mold'
-  version '1.1.1'
-  compatibility 'i686, x86_64'
+  version '615a24d1dde2e7d0756a53897ed7f46842f0c3a1'
```

Comment:
Show the first 7 letters of the commit only?

Reply:
no longer relevant. :)